guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -1,7 +1,7 @@
 [[Using_ISS]]
 == Synchronizing Content Between {ProjectServer}s
 
-{ProjectName} uses Inter-{Project} Synchronization (ISS) to synchronize content between two {ProjectServer}s including those that are airgapped.
+{ProjectName} uses {Project}'s Inter-Server Synchronization (ISS) to synchronize content between two {ProjectServer}s including those that are airgapped.
```

Comment:
```suggestion
{ProjectName} uses Inter-Server Synchronization (ISS) to synchronize content between two {ProjectServer}s including those that are airgapped.
```

In nightly, it's just `Inter-Server Synchronization (ISS)`: https://docs.theforeman.org/nightly/Content_Management_Guide/index-katello.html#Synchronizing_Content_Between_Servers_content-management

But this is fine of course.